keychain/btcwallet.go

```diff
@@ -379,13 +379,13 @@ func (b *BtcWalletKeyRing) ECDH(keyDesc KeyDescriptor,
 //
 // NOTE: This is part of the keychain.DigestSignerRing interface.
 func (b *BtcWalletKeyRing) SignDigest(keyDesc KeyDescriptor,
-	digest [32]byte) (*btcec.Signature, error) {
+	digest []byte) (*btcec.Signature, error) {
```

Comment:
The description above needs to be updated if the function were to take arbitrary bytes. However, `SignDigest`, as the name indicates, is signing a `digest`, so I would expect the `digest` parameter to be a hash digest, not arbitrary length of bytes. Plus, the `privKey.Sign` is using the underlying `signRFC6979`, which expects a hash digest.

Reply:
Makes sense. Fixed by reverting back to `[32]byte`.